tests/next_tests/integration_tests/cases.py

```diff
@@ -117,12 +117,19 @@ def from_case(
         return self
 
 
-@dataclasses.dataclass
+@dataclasses.dataclass(init=False)
 class ConstInitializer(DataInitializer):
     """Initialize with a given value across the coordinate space."""
 
     value: ScalarValue
 
+    def __init__(self, value: ScalarValue):
```

Comment:
```suggestion
    def __init__(self, value: core_defs.Scalar):
```

Reply:
and remove `ScalarValue` completely

Reply:
... but kept `ScalarValue` pointing to `core_defs.Scalar` as this seems to be the pattern that is used in this field